lms/envs/aws.py

```diff
@@ -490,6 +490,19 @@
 
 # Modules having these categories would be excluded from progress calculations
 PROGRESS_DETACHED_CATEGORIES = ['discussion-course', 'group-project', 'discussion-forum']
+PROGRESS_DETACHED_APPS = ['group_project_v2']
+for app in PROGRESS_DETACHED_APPS:
+    try:
+        app_module = __import__(app, fromlist=['app_config'])
```

Comment:
Nit: Can you use `importlib.import_module` instead of `__import__`? It's a bit nicer for something like this, and it's what edx-platform tends to use.